lib/Service/Classification/ImportanceClassifier.php

```diff
@@ -350,19 +365,10 @@
 		 * Ref https://en.wikipedia.org/wiki/F1_score
 		 */
 		$this->logger->debug("classification report: " . json_encode([
-			'recall' => $report->getRecall(),
-			'precision' => $report->getPrecision(),
-			'f1Score' => $report->getF1score(),
+			'recall' => $recallImportant,
```

Comment:
Is this correct? You use the same var twice?

Reply:
right. fixed :)